backstop/tests/date-picker.html

```diff
@@ -120,7 +120,7 @@ <h2>Date</h2>
           >
             <div class="iui-calendar-week">
               <div
-                class="iui-calendar-day iui-outside-month"
+                class="iui-calendar-day-outside-month"
```

Comment:
What is the benefit of 1 class VS modifier classes?  Seems tricky when you introduce combos like today + within range.

Reply:
Well, I would need to handle `iui-today` modifier for every different day (simple day, selected day, range day, etc) by adding a selector to it. So it increases complexity and most important specificity.